okta_rules/okta_geo_improbable_access.py

```diff
@@ -121,7 +128,14 @@ def dedup(event):
 
 def alert_context(event):
     context = okta_alert_context(event)
-    context['old_city'] = EVENT_CITY_TRACKING[event['p_row_id']].get(
-        'old_city', '<NOT_STORED>')
-    context['new_city'] = EVENT_CITY_TRACKING[event['p_row_id']]['new_city']
+    context['old_city'] = deep_get(
```

Comment:
does this need `deep_get`? wondering if it makes it longer/harder to read

Reply:
It really isn't needed here -- changed it primarily for readability